README.md

```diff
@@ -1,20 +1,25 @@
-# Notation
+# notation-go
 
 [![Build Status](https://github.com/notaryproject/notation-go/actions/workflows/build.yml/badge.svg?event=push&branch=main)](https://github.com/notaryproject/notation-go/actions/workflows/build.yml?query=workflow%3Abuild+event%3Apush+branch%3Amain)
-[![codecov](https://codecov.io/gh/notaryproject/notation-go/branch/main/graph/badge.svg)](https://codecov.io/gh/notaryproject/notation-go)
+[![Codecov](https://codecov.io/gh/notaryproject/notation-go/branch/main/graph/badge.svg)](https://codecov.io/gh/notaryproject/notation-go)
 [![Go Reference](https://pkg.go.dev/badge/github.com/notaryproject/notation-go.svg)](https://pkg.go.dev/github.com/notaryproject/notation-go@main)
 
-A collection of libraries for supporting Notation sign, verify, push, pull of oci artifacts. Based on Notary Project standard.
+notation-go offers libraries for signing, verifying, uploading, and downloading OCI artifacts using [Notary Project specifications](https://github.com/notaryproject/specifications). notation-go is used by [notation](https://github.com/notaryproject/notation) CLI for signing and verifying artifacts.
+
+notation-go reached a stable release as of July 2023 and continues to be actively developed.
+
+You can also find the Notary Project [README](https://github.com/notaryproject/.github/blob/main/README.md) to learn about the overall Notary Project.
+
+> **Note** The Notary Project documentation is available [here](https://notaryproject.dev/docs/). 
 
 ## Table of Contents
-- [Core Documents](#core-documents)
+- [Documentation](#documentation)
 - [Code of Conduct](#code-of-conduct)
 - [License](#license)
+ 
+## Documentation
 
-## Core Documents
-
-* [Governance for Notation](https://github.com/notaryproject/notary/blob/master/GOVERNANCE.md)
```

Comment:
We need to update Code of conduct section (commented here as I cannot comment on section which has no diff).

Reply:
Sorry I didn't see the where should be updated in the Code of conduct section. Those information are still up to date. Can you point it out, or raise another PR to update it?